examples/cpp/CMakeLists.txt.cpp_example.in

```diff
@@ -24,9 +24,17 @@ find_package(@PRODUCT_NAME_NOSPACE@ REQUIRED COMPONENTS cpp)
 
 add_executable(${PROJECT_NAME} @ADD_EXAMPLE_SOURCE@)
 
+configure_file("startserver.ps1" ${CMAKE_CURRENT_BINARY_DIR} COPYONLY)
+configure_file("stopserver.ps1" ${CMAKE_CURRENT_BINARY_DIR} COPYONLY)
 configure_file("startserver.sh" ${CMAKE_CURRENT_BINARY_DIR} COPYONLY)
 configure_file("stopserver.sh" ${CMAKE_CURRENT_BINARY_DIR} COPYONLY)
 
+configure_file("README.md" ${CMAKE_CURRENT_BINARY_DIR} COPYONLY)
+
+if(WIN32)
+  configure_file(../../../bin/apache-geode.dll ${CMAKE_CURRENT_BINARY_DIR}/Debug/apache-geode.dll COPYONLY)
```

Comment:
This won’t work if I make a copy of the examples directory. Use findGeode and copy the dll.

Reply:
Good call. Changed to using the find module for geode-native to locate the dll.